website/versioned_docs/version-5.x/module-types.md

```diff
@@ -0,0 +1,51 @@
+---
+id: module-types
+title: single-spa microfrontend types
+sidebar_label: Microfrontend Types
+---
+
+# Concept: single-spa microfrontend types
+
+Single-spa has [different categories](/docs/microfrontends-concept#types-of-microfrontends) of microfrontends. It is up to you where and how you use each of them. However, the single-spa core team has [recommendations](/docs/recommended-setup/#applications-versus-parcels-versus-utility-modules).
```

Comment:
Oh one thing I just remembered - we should start using relative links for linking between files. This is important for versioning - we want to always stay within the same version of the single-spa docs when linking. See related https://github.com/single-spa/single-spa.js.org/issues/178 and https://v2.docusaurus.io/docs/versioning/#linking-files

```suggestion
Single-spa has [different categories](/docs/microfrontends-concept#types-of-microfrontends) of microfrontends. It is up to you where and how you use each of them. However, the single-spa core team has [recommendations](./recommended-setup.md#applications-versus-parcels-versus-utility-modules).
```